specifications/sep-payment-channel-transactions.md

```diff
@@ -229,45 +226,58 @@ payments to one another within the channel.
+disburse $8 to I and $22 to R.
 
 To update the payment channel state, the participants:
 
 1. Increment i.
 2. Sign and exchange a closing transaction C_i.
-3. Sign and exchange a declaration transaction D_i.
+3. Sign and exchange zero or more payment transactions P_{i,a}.
+4. Sign and exchange a declaration transaction D_i.
 
-It is important that D_i is signed after C_i because D_i will invalidate any
-previously signed C_i. If I and R signed and exchanged D_i first either party
-could prevent the channel from closing without coordination by submitting D_i
-and refusing to sign C_i.  The participants would not be able to close the
-channel, or regain control of the accounts, and the assets within without
-coordinating with each other.
+It is important that D_i is signed after C_i and P_i because D_i will invalidate
+any previously signed C_i and P_i. If I and R signed and exchanged D_i first
+either party could prevent the channel from closing without coordination by
+submitting D_i and refusing to sign C_i.  The participants would not be able to
+close the channel, or regain control of the accounts, and the assets within
+without coordinating with each other.
 
 The transactions are constructed as follows:
 
-- C_i, the _closing transaction_, disburses funds from EI to ER and/or from ER
-to EI, and changes the signing weights on EI such that I unilaterally controls
-EI, and the signing weights on ER such that R unilaterally controls ER.  C_i has
-source account EI, sequence number s_i+1, a `minSeqAge` of O (the observation
-period time duration), and a `minSeqLedgerGap` of O (the observation period
-ledger count).
+- C_i, the _closing transaction_, changes the signing weights on EI such that I
+unilaterally controls EI, and the signing weights on ER such that R unilaterally
+controls ER.  C_i has source account EI, sequence number s_i+1+n.
 
   The `minSeqAge` and `minSeqLedgerGap` prevents a misbehaving party from
   executing C_i when the channel state has already progressed to a later
-  iteration number, as the other party has the period O to invalidate C_i by
+  iteration number, as the other party has the period 2O to invalidate C_i by
   submitting D_i' for some i' > i.
   
   C_i contains operations:
-  - One `PAYMENT` operation for each trustline that is disbursing funds from EI
-  to ER, or from ER to EI.
   - One or more `SET_OPTIONS` operation adjusting escrow account EI's thresholds
   to give I full control of EI, and removing R's signers.
   - One or more `SET_OPTIONS` operation adjusting reserve account ER's
   thresholds to give R full control of ER, and removing I's signers.
 
+- P_{i,a}, the _payment transaction_, disburses a single asset a from EI to ER
+or from ER to EI. P_{i,a} has source account EI, sequence number s_i+1+a, where
+a is the zero index of the asset where each asset in the channel is assigned an
+index. P_{i,0}, the first payment, is assigned a `minSeqAge` of O (the
```

Comment:
"O" defining both `observation period time duration` and `the observation period ledger count` is confusing to me since it's implying that "O" would be the same value for both (which I don't think is the case). It looks like we're using "O" generally in this SEP to describe "observation period", which is describing two fields.

💭  proposal to change "O" here to `O_minSeqAge` and `O_minSeqLedgerGap`, or some other way of differentiating between its 2 uses

Reply:
Agree it is a little confusing, although I'm not sure there's a good formal notation like what you propose. The correct way to represent the two different values would be to define the separately, such as to define them as OA and OG. I'm not sure it is worth introducing two separate variables though. It's out of scope of this PR. Could you open an issue?

Reply:
https://github.com/stellar/experimental-payment-channels/issues/79